### ISSUE TITLE
Env var interpolation in name and namespace

### ISSUE THREAD

**Issue by danielcb** [roboll/helmfile#86]
Hi,
since it was done in #36 for repositories: Would it be possible to extend the interpolation to name and namespaces for automated deployments?

For namespace it looks like it can be easily extended in flagsForRelease(). For name it's currently scattered between SyncReleases and DiffReleases.. but generally just wondering: Any reason why it's not (yet) possible / why it should not be possible for those two fields?

**Comment by danielcb** [comment]
Added a PR to illustrate what I was trying to say/ask.

**Comment by manics** [comment]
Could this be extended to other (or all?) fields? I'm currently using YAML anchors and references to deploy the same chart with multiple configurations, e.g.
```
RELEASE_COMMON: &RELEASE_COMMON
  chart: jupyterhub/jupyterhub
  version: v0.7-4104389

releases:

  - <<: *RELEASE_COMMON
    name: jupyterhub-1
    namespace: jupyterhub-1
    ...
  - <<: *RELEASE_COMMON
    name: jupyterhub-2
    namespace: jupyterhub-2
    ...
```
But interpolating chart and version would be more readable.

**Comment by danielcb** [comment]
Yes. Which fields? Chart/version?

**Comment by manics** [comment]
Those are the two I'm interested in, but for consistency perhaps it's easier to say which fields shouldn't be interpolated?

**Comment by mumoshu** [comment]
@manics @danielcb I rather want to start by limiting helmfile's dynamism to be necessary and sufficient for intended use-cases, so that hopefully helmfile.yml doesn't get mess.
So, would you mind sharing us your exact use-cases of wanting templates everywhere for further discussion? Thx!

**Comment by mumoshu** [comment]
Anyway, closing this as the original issue is resolved via #90 and #87. Please feel free to submit separate feature requests for another places you want templating!

**Comment by cmeury** [comment]
Somehow I think using YAML anchors is not that bad. Just using the capabilities of the format.

But I had a similar request in #59, using a `defaults` section. How do you think these two compare?
